components/playback/controls/player/AddCustomer.vue

```diff
@@ -8,7 +8,6 @@
       <InputWidget
         :title_label="'Enter Alphanumeric ID'"
```

Comment:
The alphanumeric ID and API key should still have `spellcheck='false'`.  Those are not fields expected to contain English words.  Just the Nickname should use spellchecking

Reply:
ok.